libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartSyncPlugin/SFSmartSyncPlugin.m

```diff
@@ -52,26 +51,25 @@ @interface SFSmartSyncPlugin ()
 
 @implementation SFSmartSyncPlugin
 
-- (SFSmartSyncSyncManager *)syncManager
+- (SFSmartSyncSyncManager *) syncManager
 {
     return [SFSmartSyncSyncManager sharedInstanceForStore:[SFSmartStore sharedStoreWithName:kDefaultSmartStoreName]];
 }
 
-- (SFSmartSyncSyncManager *)globalSyncManager
+- (SFSmartSyncSyncManager *) globalSyncManager
 {
     return [SFSmartSyncSyncManager sharedInstanceForStore:[SFSmartStore sharedGlobalStoreWithName:kDefaultSmartStoreName]];
 }
 
-- (void)resetSyncManager
+- (void) resetSyncManager
```

Comment:
We can't. We need it exposed via the plugin. Also, it's not purely for unit testing. It can be used to reset the instance of ```SyncManager``` too.